hs_access_control/management/commands/community.py

```diff
@@ -0,0 +1,322 @@
+    print("              remove: remove the group from the community.")
+    print("          Options for group metadata update include:")
+    print("              --prohibit_view: don't allow viewing of this group's resources.")
+    print("              --allow_edit: allow this group to edit other groups' resources.")
+    print("      owner {oname} {request}: owner commands")
+    print("      owner {oname} {request}: owner commands")
+    print("          {oname}: owner name.")
+    print("          {request} is one of:")
+    print("              [blank]: list community owners")
+    print("              add: add an owner for the community.")
+    print("              remove: remove an owner from the community.")
+
+
+class Command(BaseCommand):
+    help = """Manage communities of groups."""
+
+    def add_arguments(self, parser):
+
+        # a command to execute
+        parser.add_argument('command', nargs='*', type=str)
+
+        # Named (optional) arguments
+        parser.add_argument(
+            '--prohibit_view',
+            action='store_true',            # True for presence, False for absence
+            dest='prohibit_view',           # value is options['prohibit_view']
+            help="prohibit viewing of group's resources by community",
+        )
+
+        parser.add_argument(
+            '--allow_edit',
+            action='store_true',            # True for presence, False for absence
+            dest='allow_edit',              # value is options['allow_edit']
+            help="allow group to edit other groups' resources in community",
+        )
+
+        parser.add_argument(
+            '--owner',
+            dest='owner',
+            help='owner of community (does not affect quota)'
+        )
+
+        parser.add_argument(
+            '--description',
+            dest='description',
+            help='description of community'
+        )
+
+        parser.add_argument(
+            '--purpose',
+            dest='purpose',
+            help='purpose of community'
+        )
+
+    def handle(self, *args, **options):
+
+        if len(options['command']) > 0:
+            cname = options['command'][0]
+        else:
+            cname = None
+
+        if len(options['command']) > 1:
+            command = options['command'][1]
+        else:
+            command = None
+
+        # resolve owner: used in several update commands as grantor
+        if options['owner'] is not None:
+            oname = options['owner']
+        else:
+            oname = 'admin'
+
+        owner = user_from_name(oname)
+        if owner is None:
+            usage()
+            exit(1)
+
+        if options['allow_edit']:  # this is a group privilege
+            privilege = PrivilegeCodes.CHANGE
+        else:
+            privilege = PrivilegeCodes.VIEW
+
+        # not specifing a community lists active communities
+        if cname is None:
+            print("All communities:")
```

Comment:
Should usage be printed here as well?

Reply:
Well, I don't think so..... it is a correct usage to list all communities.